schemas/rules/operators/comparison/in.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://eq.ons.gov.uk/rules/operators/comparison/in.json",
+  "description": "Evaluates if a value is in an array, takes two arguments",
+  "type": "object",
+  "properties": {
+    "in": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": [
+        {
+          "$ref": "https://eq.ons.gov.uk/rules/definitions.json#/string_value"
+        },
+        {
+          "$ref": "https://eq.ons.gov.uk/rules/definitions.json#/array_value"
+        }
+      ]
+    }
```

Comment:
Shouldn't the first value be `any_value` as it could be other types like a number and also the current definition wouldn't support values from other sources like metadata. I know `any_value` could be an array, but I still think that is fine to accept for this case.

Reply:
To address this I have created a `any_value_except_array` in https://github.com/ONSdigital/eq-questionnaire-validator/pull/96/commits/2c073ca22ce6e90ac03b8b4e943c6bf121e3a048.